### ISSUE TITLE
German translation problems

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#722]
```
by wiese.hano

* "Spielstand überschrieben?" should be changed into "Spielstand 
überschreiben?".
* "über mäßig" should be changed into "höher" (for one step above moderate).
* Some dialogs between the player and the enemy captains of hostile fleets at 
the beginning of realtime fights, still are in english, thought they shoud be 
in german.

```

Original issue reported on code.google.com by `akarn...@gmail.com` on 1 Apr 2013 at 5:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 2 Jun 2013 at 11:06
- Added labels: **Milestone-0.96**

**Comment by GoogleCodeExporter** [comment]
```
The first and third were fixed in issue #825. I'm not sure about "über 
mäßig" as there is a "Hoch" setting already, which might confuse things.
```

Original comment by `akarn...@gmail.com` on 29 Jan 2014 at 11:51
- Changed state: **Fixed**
